### PR TITLE
Added Make Adiabatic by Name

### PR DESCRIPTION
**Pull request by MingboPeng**
Modified from Make Adiabatic.
Use this component to modify the specific surface to adiabatic without decompose the zones

### CODE REVIEW

File: src/Honeybee_Make Adiabatic by Name.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
+# 
+# This file is part of Honeybee.
+# 
+# Copyright (c) 2013-2017, Mostapha Sadeghipour Roudsari <mostapha@ladybug.tools>
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Make Adiabatic by Name.py]: @MingboPeng  this should change to your name and email.

File: src/Honeybee_Make Adiabatic by Name.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ghenv.Component.IconDisplayMode = ghenv.Component.IconDisplayMode.application
+ghenv.Component.Category = "Honeybee"
+ghenv.Component.SubCategory = "10 | Energy | Energy"
+#compatibleHBVersion = VER 0.0.56\nNOV_04_2016
+#compatibleLBVersion = VER 0.0.59\nFEB_01_2015
+try: ghenv.Component.AdditionalHelpFromDocStrings = "0"
+except: pass
+
+import scriptcontext as sc
+import uuid
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Make Adiabatic by Name.py]: you don't need `uuid` for this component.

File: src/Honeybee_Make Adiabatic by Name.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    hb_hive = sc.sticky["honeybee_Hive"]()
+    HBObjects = hb_hive.callFromHoneybeeHive(HBObjs)
+    
+    for HBO in HBObjects:
+        if HBO.objectType == "HBZone":
+            for HBS in HBO.surfaces:
+                if HBS.name in nameList:
+                    HBS.BC = "Adiabatic"
+        else:
+            if HBO.name in byName:
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Make Adiabatic by Name.py]: This is a bug! `byName` should be changed to `nameList`.

**Comment by mostaphaRoudsari** [comment]
@MingboPeng thanks! this is a very useful component. I added a couple of comments to your code. We can merge this in once you get them fixed.

File: src/Honeybee_Make Adiabatic by Name.py (hunk 3 above)
Comment (MingboPeng) [inline src/Honeybee_Make Adiabatic by Name.py]: @mostaphaRoudsari  Ha, thanks.

**Comment by mostaphaRoudsari** [comment]
Merged! Thanks @MingboPeng.

**Pull request merged**